cf-oscore/src/main/java/org/eclipse/californium/oscore/Encryptor.java

```diff
@@ -198,7 +217,17 @@ protected static Encrypt0Message prepareCOSEStructure(byte[] confidential) {
 		} else {
 			bRes.write(firstByte);
 		}
-		
+
+		//Encode the Context ID length and value if to be included
+		if(ctx.getIncludeContextId() == true) {
```

Comment:
Please insert a space (' ') between "if" and "(".

Reply:
Fixed in commit c2bffa2b0b1144b65108adbd7f1cc08638928d44